README.md

```diff
@@ -1,15 +1,59 @@
-# Team members present: 
-## What's this PR do? 
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] Styling
-- [ ] Testing
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-## Where should the reviewer start? 
-## How should this be manually tested? 
-## Any background context you want to provide? 
-## What are the relevant tickets? 
-* Addresses:
-* Closes:
-## Screenshots (if appropriate) 
-## Questions:
\ No newline at end of file
+## House Rules
+
+**Contributors**
+
+[Hanna Kim](https://github.com/hannakim91), [Michael Walker](https://github.com/MichaelEWalker87), [Katy St Sauveur](https://github.com/krogowsk531)
+
+**Project Description**
+
+*A site for disputing game rules where a user can select a game, get information about that game and fill out a form that allows them to dispute a game rule. The rule dispute will persist through local storage in each individual game page so that a user can debate rules for the specified game.
+
+**Installation**
+
+**Fork this repository:**
+
+* https://github.com/krogowsk531/house-rules
+
+**Clone your forked repository**
+
+* `git clone` and the copied URL
+
+**Change into the directory and install the project dependencies**
+
+* `cd` into directory and run `npm install` for dependencies
+
+**Project Goals**
+
+* Create a functioning app while learning a new technology
+* Decide what the most viable product for the app will be
+* Use an external API of the teams choosing
+* The data from the API must be able to be manipulated by the user
```

Comment:
I don't think we have a way for users to manipulate the data directly to the api (only running GET requests) but we are having user interaction saved locally

Reply:
I removed the confusing line.  I think that we were accomplishing the goal, however.  In the spec it says:

"The application cannot simply be a display of data - there needs to be some way for the user to work with or manipulate the data (think favoriting at the very least, but be more creative than favoriting)"